app/api/settings/settings.js

```diff
@@ -82,9 +84,15 @@ function removeTemplate(filters, templateId) {
   });
 }
 
+function setDefaults(storedSettings) {
+  const [first] = storedSettings;
```

Comment:
`[first]` is just `[settings]`.  MongoDB works with array of documents, but we are using just a single document.  So it's not a 'first' settings, as suggested by the naming, the first document contains the ONLY settings, so I think just naming them 'settings' is enough.

Reply:
👍